OpenRA.Game/FileFormats/HvaReader.cs

```diff
@@ -43,13 +46,20 @@ public HvaReader(Stream s)
 
 					for (var k = 0; k < 12; k++)
 						Transforms[c + ids[k]] = s.ReadFloat();
+
+					var testMatrix = new float[16];
```

Comment:
Can you declare this outside the loop so we don't create a bunch of throwaway arrays? (Sorry for missing this on the first review)